packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -22,13 +22,13 @@ const RESPONSIVE_MAX_CONSTRAINT = [479, 639, 1023, 1365, 1919, 99999999];
 /**
  * User specified mode to default to, useful for server side rendering scenarios.
  */
-let _defaultMode: ResponsiveMode | undefined;
+export let _defaultMode: ResponsiveMode | undefined;
```

Comment:
Should have said this before but I'm not sure that exporting these directly from the file is actually correct/desirable. For one thing, this is very much an internal detail that shouldn't really be exposed. Also, I'm pretty sure that when you import it the value will be *copied* and not pick up or propagate any updates.

Probably the easiest solution would be to add a new function `getInitialResponsiveMode` which returns `_defaultMode || _lastMode || ResponsiveMode.large`, and use it in both the new and old verisons.